plugins/obs-outputs/flv-mux.c

```diff
@@ -59,22 +66,32 @@ enum datatype_t {
 static void s_w4cc(struct serializer *s, enum video_id_t id)
 {
 	switch (id) {
+	case CODEC_NONE:
+		assert(0 && "Tried to serialize CODEC_NONE");
```

Comment:
We set `NDEBUG` for all configurations apart from `Debug`, so the assert is inert in those. Is that taken into account here?

Reply:
I briefly checked with @notr1ch off-thread and it seems that assert here is okay. @palana could you summarize the intent with the assert here to make sure we're all on the same page?

Reply:
This is fine as-is for now then. If need be, we'll re-evaluate later.